gradle/build-docs.gradle

```diff
@@ -9,211 +9,78 @@ buildscript {
 	dependencies {
 		classpath 'io.spring.gradle:docbook-reference-plugin:0.3.0'
 		classpath 'org.asciidoctor:asciidoctor-gradle-plugin:1.5.0'
-		classpath 'org.ajoberstar:gradle-git:0.8.0'
+		//classpath 'org.asciidoctor:asciidoctorj-pdf:1.5.0-alpha.6' // asciidoctor pdf still has issues with callouts, etc. Still relying on docbook+pdf for now
```

Comment:
Wanted to leave that here reminding the next reader about
- the fact that asciidoctor now has direct support for pdf...
- ... but it's not ready yet